src/In.ProjectEKA.HipLibrary/Patient/Model/LinkConfirmationRepresentation.cs

```diff
@@ -12,6 +12,9 @@ public class LinkConfirmationRepresentation
 
         public IEnumerable<CareContextRepresentation> CareContexts { get; }
 
+        public LinkConfirmationRepresentation()
+        {
+        }
```

Comment:
Why do we need this?

Reply:
For sending the emty response. Else that field is missed when it is null

Reply:
that's just serializer settings.

options.JsonSerializerOptions.IgnoreNullValues = false;

Startup.cs in 116

Always Try not to make anything mutable. Make sure nothing breaks after this settings change.